pkg/kube/log_tail.go

```diff
@@ -20,6 +20,8 @@ import (
 	"strings"
 )
 
+const LogTailDefaultLength = 10
```

Comment:
```suggestion
const logTailDefaultLength = 10
```
No need to export unless required